content/en/docs/Reference/veneers.md

```diff
@@ -161,6 +161,65 @@ schema: olm.bundle
 ```
 </details>
 
+#### Converting from FBC to Basic Veneer
+Operator authors can convert a File-Based Catalog (FBC) to a basic veneer by processing the output of the `opm render` command through either `jq` or `yq`.
+
+To convert a File-Based Catalog to a Basic Veneer using `jq`, run the following command:
+
+```sh
+opm render <catalogRef> -o json | jq 'if (.schema == "olm.bundle") then {schema: .schema, image: .image} else . end'
+```
+
+Example veneer in JSON format after the conversion:
+
+```json
+{
+  "schema": "olm.package",
+  "name": "hello-kubernetes",
+  "defaultChannel": "alpha",
+  "description": "hello-kubernetes"
+}
+{
+  "schema": "olm.channel",
+  "name": "alpha",
+  "package": "hello-kubernetes",
+  "entries": [
+    {
+      "name": "hello-kubernetes.v0.0.1"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "image": "docker.io/test/hello-kubernetes-operator-bundle:v0.0.1"
+}
+```
+
+To convert a File-Based Catalog to a Basic Veneer using `yq`, run the following command:
+
+```sh
+opm render <catalogRef> -o yaml | yq eval -i 'select(.schema == "olm.bundle") = {"schema": .schema, "image": .image}' test.yaml - test.yaml
```

Comment:
Is `test.yaml - test.yaml` intentional on this line?

Reply:
@grokspawn  Yes, this is intentional. It's just the way piping seems to work with `yq`. I was pretty confused as well! 
I thought `opm render <catalogRef> -o yaml | yq eval -i 'select(.schema == "olm.bundle") = {"schema": .schema, "image": .image}' test.yaml` should work but it ended up with the file having no content at all.